src/spikeinterface/preprocessing/tests/test_filter_gaussian.py

```diff
@@ -21,7 +21,7 @@ def test_filter_gaussian():
     recording.annotate(is_filtered=True)
     recording = recording.save(folder=cache_folder / "recording")
 
-    rec_filtered = gaussian_bandpass_filter(recording)
+    rec_filtered = gaussian_filter(recording)
 
```

Comment:
Maybe we can add tests for lowpass and highpass filters?

Reply:
yes.. but I'm not sure what the tests should look like

Reply:
I guess you can create white noise (each point in time is independent and drawn from a Gaussian), and we can check that after a lowpass/highpass filter, the high/low frequencies are removed (while low/high are unaffected)?